app/controllers/api/v2/ping_controller.rb

```diff
@@ -1,15 +1,20 @@
 module Api
   module V2
     class PingController < BaseController
+      resource_description do
+        api_version 'v2'
+        api_base_url ''
+      end
+
       skip_before_action :authorize, only: [:ping]
 
-      api :GET, '/ping', N_("Shows status of Foreman system and it's subcomponents")
+      api :GET, '/api/ping', N_("Shows status of Foreman system and it's subcomponents")
```

Comment:
Will this still be available under `/api/v2/ping` as well?

Reply:
Sure, I've checked it with `curl`. Apipie-rails doesn't affect the routes specified in `routes`.